src/cli/services/make_service.ts

```diff
@@ -0,0 +1,14 @@
+const encoder = new TextEncoder();
+
+export const testTemplate: Uint8Array = encoder.encode(
+  `import { Rhum } from "https://deno.land/x/rhum@v1.1.4/mod.ts";
```

Comment:
needs to be references in the update versions script

Reply:
d5eca1bb

Reply:
6e966a716bda53

Reply:
I think we have update Deno versions script for the bumper, so when our deps are updated we update the references too eg Deno version in workflows, then we have bump versions script which is for the pre release, to update the Rhum versions ready for a new release

So might need to move this into the bump versions script - I did add that bump versions thing to services but I believe we could abstract the getting of the versions from the args into that services file, not sure how tho

Reply:
i did find an issue (via tests) with the regex i passed in. it's fixed now though. here's the file:

https://github.com/drashland/rhum/pull/80/files#diff-2e202eda61bc9862e1f12e66fe6fcbbf971e0da38fb65bbaf846eade32aa6116R22-R28